### PR TITLE
[yaireo__tagify] persisted data, bump verion 4.9

### PR DESCRIPTION
**Pull request by Brakebein**
Please fill in this template.

- [x] Use a meaningful title for the pull request. Include the name of the package modified.
- [x] Test the change in your own code. (Compile and run.)
- [x] [Add or edit tests](https://github.com/DefinitelyTyped/DefinitelyTyped/blob/master/README.md#my-package-teststs) to reflect the change.
- [x] Follow the advice from the [readme](https://github.com/DefinitelyTyped/DefinitelyTyped/blob/master/README.md#make-a-pull-request).
- [x] Avoid [common mistakes](https://github.com/DefinitelyTyped/DefinitelyTyped/blob/master/README.md#common-mistakes).
- [x] [Run `npm test <package to test>`](https://github.com/DefinitelyTyped/DefinitelyTyped/blob/master/README.md#running-tests).

If changing an existing definition:
- [x] If this PR brings the type definitions up to date with a new version of the JS library, update the version number in the header.
- [x] Provide a URL to documentation or source code which provides context for the suggested changes:
  - Release [v4.9.0](https://github.com/yairEO/tagify/releases/tag/v4.9.0)
  - Persisted data: https://github.com/yairEO/tagify/commit/8b44f9c6680fd56656e43d90d998c5ca0220489e
  - DT issue/discussion: #56529 #56771

### CODE REVIEW

File: types/yaireo__tagify/index.d.ts [hunk 1]
```diff
@@ -1602,6 +1608,31 @@ declare class Tagify<T extends Tagify.BaseTagData = Tagify.TagData> {
      */
     setDisabled(disabled: boolean): void;
 
+    /**
+     * Get data for the specific instance by `key` parameter from `localStorage`.
+     * @param key `localStorage` key (under the tagify namespace).
+     * @returns Data stored under `key` in `localStorage`.
+     * Returns `undefined` if {@link TagifyConstructorSettings.id} has not been set or no entry exists for `key` in `localStorage`.
+     */
+    getPersistedData(key: string): any;
```
Comment (blutorange) [inline types/yaireo__tagify/index.d.ts:1617]: Can you change the return type to `unknown`? `any` should be avoided if possible, it easily pollutes other variables and effectively turns off type checks.

File: types/yaireo__tagify/index.d.ts [hunk 2]
```diff
@@ -1602,6 +1608,31 @@ declare class Tagify<T extends Tagify.BaseTagData = Tagify.TagData> {
...
+
+    /**
+     * Set data for the specific instance.
+     * Must supply a second parameter which will be the key to save the data in the `localStorage`
+     * (under the tagify namespace).
+     * In order to use this method, {@link TagifyConstructorSettings.id} must be set.
+     * @param data Data to store in `localStorage`.
+     * @param key `localStorage` key (under the tagify namespace).
+     */
+    setPersistedData(data: any, key: string): void;
```
Comment (blutorange) [inline types/yaireo__tagify/index.d.ts:1627]: Same as above, for consistency `data` should be of type `unknown` (although it doesn't matter much for parameters)

**Review by blutorange — CHANGES_REQUESTED** [review]
Would be great if you could change `any` to `unknown`. Other than that, looks good me : )

**Review by blutorange — APPROVED** [review]
Looks ready to be merged!

**Comment by Brakebein** [comment]
Ready to merge

**Pull request merged**